llmfoundry/models/layers/ffn.py

```diff
@@ -139,24 +140,27 @@ def __init__(
             expansion_ratio,
             ffn_hidden_size,
         )
-        self.fc_kwargs: dict[str, Any] = {
-            'bias': bias,
-        }
 
-        self.fc_kwargs['device'] = device
+        # Usually, fc_type dict should be passed in through MPTBlock's __init__ function.
+        if fc_type is None:
+            fc_type = fc_type_defaults
```

Comment:
should be a deepcopy of the defaults right? to avoid the same issue of accidentally modifying the defaults dict for everyone

Reply:
applies throughout

Reply:
yeah i guess we want to avoid a situation where someone accidentally modifies the dict and it ripples throughout the model...

Reply:
fixed